### PR TITLE
Update Docker images

### PR DESCRIPTION
**Pull request by mnmkng**
Deprecated old docker images and replaced them with new ones. Not sure if this will work out of the box, because previously, `puppeteer` was always included in `dependencies` of the SDK, but now, users need to add it to their `package.json`. Are there different `package.json` files for different images? We need to add `puppeteer` to some and `playwright` to other.

### CODE REVIEW

**Review by drobnikj — CHANGES_REQUESTED** [review]
The problem can be with a single file. We are installing packages(npm install package1 package2) matches from all `require('')` in code. If the user didn't require playwrite or any other requested package for the docker image the run will fail.

**Comment by drobnikj** [comment]
Wait, maybe I'm not right as we already install these packages in the base image as they are listed there as dep e.g. https://github.com/apify/apify-actor-docker/blob/master/node-playwright-chrome/Dockerfile.
I took it back, it can work this way.

**Review by drobnikj — APPROVED** [review]

File: src/consts.js [hunk 1]
```diff
@@ -68,54 +68,109 @@ export const META_ORIGINS = {
...
+        displayName: 'Node.js 14 + Playwright + All Browsers on Ubuntu',
         copyChown: 'myuser:myuser',
     },
 
     // Deprecated:
-    // TODO: Keep the for some time and then migrate acts to recommended images.
+    // These are here because we made breaking changes in the client that could break existing single file actors.
+    // We will get a rid of this along with the whole single file logic.
     {
-        name: 'apify/actor-node-puppeteer',
```
Comment (mtrunkat) [inline src/consts.js:107]: Shouldn't we keep these just renamed to deprecated? Otherwise, the building will break if somebody use it.

**Review by mtrunkat — CHANGES_REQUESTED** [review]

File: src/consts.js (hunk 1 above)
Comment (mnmkng) [reply src/consts.js:107]: `apify/actor-node-puppeteer` images were deprecated 3 years ago. I think it's safe to remove them.

Comment (jancurn) [reply src/consts.js:107]: How about a quick query to see if somebody is still building with them?

Comment (mnmkng) [reply src/consts.js:107]: I did the query and there were only some free users using it with non-production actors. It seems that they stumbled upon the image in the list and were trying it out. => Better remove it.

**Pull request merged**